tests/Fixer/Import/OrderedImportsFixerTest.php

```diff
@@ -780,7 +780,7 @@ public function provide70Cases()
     |--------------------------------------------------------------------------
     | Test sorting by length
     |--------------------------------------------------------------------------
-    */
+     */
```

Comment:
I would align the stars only if whole comment block is based on stars, like
```
/*
 *
   *
*
         */
```,
not mixed up with different things, like `|`.

WDYF?